### ISSUE TITLE
What is the accuracy of this method?

### ISSUE THREAD

**Issue by gmcalo** [humanpose1/KPConvTorch#3]
Could you provide your training result?

**Comment by humanpose1** [comment]
Hi, I could not reproduce the results of the original paper (~86% accuracy on modelNet) Also, my current implementation is slower. If you want a better implementation of KPConv in Pytorch, you should have a look at [this repo](https://github.com/nicolas-chaulet/torch-points3d).
